src/components/Tool/FixCommitmentInfo/GetInvalidStatuses.graphql

```diff
@@ -0,0 +1,15 @@
+query GetInvalidStatuses($accountListId: ID!) {
+  contacts(
+    accountListId: $accountListId
+    contactsFilter: { statusValid: false }
```

Comment:
The current version applies the deceased filter also
```suggestion
    contactsFilter: { statusValid: false, deceased: false }
```

Although it's not in GraphQL yet. I messaged @soberstadt.